Update randomness-onchain.mdx - how to use the Random object (#17386)
MystenLabs/sui: docs/content/guides/developer/advanced/randomness-onchain.mdx

}
```

`Random` has a reserved address `0x8`. See <a href="/references/framework/sui-framework/random" data-noBrokenLinkCheck='true'>random.move</a> for the Move APIs for accessing randomness on Sui.

:::note

> Although `Random` is a shared object, it is inaccessible for mutable operations, and any transaction attempting to modify it will be unsuccessful.

Reviewer: ```suggestion
Although `Random` is a shared object, it is inaccessible for mutable operations, and any transaction attempting to modify it fails.
```

- The number of new objects.
- The number of objects that can be used (including dynamic fields).

## Accessing `Random` from TypeScript

> If one wants to call `roll_dice(r: &Random, ctx: &mut TxContext)` in module `example`, the following code snippet can be used:

Reviewer: ```suggestion
If you want to call `roll_dice(r: &Random, ctx: &mut TxContext)` in module `example`, use the following code snippet:
```
